src/Uno.UI/UI/Xaml/Controls/Primitives/VisualStatesHelper.mux.cs

```diff
@@ -10,31 +10,31 @@ internal static IEnumerable<string> GetValidVisualStatesListViewBaseItem(ListVie
 		{
 			int index = 0;
 			//int expectedVisualStatesSize = 6;
-			int expectedVisualStatesSize = 1; // See Uno TODO below
+			int expectedVisualStatesSize = 2; // See Uno TODO below
```

Comment:
Is there still a todo?

Reply:
Yes, the remaining states need to be added later as well, but probably when we fully port list items and selector items code. The final size should be 6 👍